src/integrationTest/java/uk/gov/hmcts/darts/event/service/impl/StandardEventHandlerTest.java

```diff
@@ -34,39 +39,49 @@ class StandardEventHandlerTest extends IntegrationBaseWithGatewayStub {
     @Autowired
     EventDispatcher eventDispatcher;
 
+
+    @MockBean
+    private UserIdentity mockUserIdentity;
+
+    @BeforeEach
+    public void setupStubs() {
+        UserAccountEntity testUser = dartsDatabase.getUserAccountStub().getIntegrationTestUserAccountEntity();
+        when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
+    }
+
     @Test
     void throwsOnUnknownCourthouse() {
         dartsDatabase.save(someMinimalCase());
         DartsEvent dartsEvent = someMinimalDartsEvent().courthouse(UNKNOWN_COURTHOUSE);
         dartsEvent.setCaseNumbers(List.of("123"));
         dartsEvent.setDateTime(today);
         assertThatThrownBy(() -> eventDispatcher.receive(dartsEvent))
-            .isInstanceOf(DartsApiException.class);
+                .isInstanceOf(DartsApiException.class);
```

Comment:
loads of formatting changes here? Can we clean these up hard to see the wood for the trees

Reply:
formatting fixed